### PR TITLE
Alok/integration schedule scripts to run per hour

### PR DESCRIPTION
**Pull request by aloka-cuelogic**
Integrated Ganesh, Deepali Code

Schedule a celery job that run every hours and update record in database  
Made changes for celery job in settings.py file

Updated requirement.txt file for same.

### CODE REVIEW

**Comment by aloka-cuelogic** [comment]
@sameetb-cuelogic @onkartibe 

Please review and merge this PR.
We have completed the dev task for scrapping POC.

File: requirement.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
+amqp==1.4.6
```
Comment (onkartibe) [inline requirement.txt:1]: 2 files are there for requirement
1. requirement.txt
2. requirements.txt

please keep the actual one.

also from requirement.txt are we using all the packages?

Comment (aloka-cuelogic) [inline requirement.txt:1]: Removed Duplicate File.
Ya we are using all the packages

**Comment by onkartibe** [comment]
Celery Documentation:

https://docs.google.com/document/d/1_OzHcOjbe3leo9hClsMV-oYUCozp4g0r43FelVuv2D4/edit

**Pull request merged**